### ISSUE TITLE
Did you implement CIoU-loss?

### ISSUE THREAD

**Issue by AlexeyAB** [Tianxiaomo/pytorch-YOLOv4#124]
@Tianxiaomo Hi!

Did you implement CIoU-loss?

Examples: 

* C: https://github.com/AlexeyAB/darknet/blob/a71b9a6e9a009cf94900c53deb344c5204835700/src/box.c#L233-L256

* Matlab: https://github.com/Zzh-tju/DIoU/blob/master/simulation%20experiment/dCIOU.m

* Python: https://github.com/VCasecnikovs/Yet-Another-YOLOv4-Pytorch/blob/2e18612e1852abbf35b4dac55a00f2a3b2d814ed/model.py#L527-L561

* Python: https://github.com/ultralytics/yolov3/blob/eca5b9c1d36e4f73bf2f94e141d864f1c2739e23/utils/utils.py#L262-L282

Desctiption: https://medium.com/@jonathan_hui/yolov4-c9901eaa8e61

![1_egnHdTEZcYJKkcPkDTgaVw](https://user-images.githubusercontent.com/4096485/85800301-626c9400-b749-11ea-93df-6d323f55ce71.jpeg)

**Comment by Tianxiaomo** [comment]
Thank you for your reference, the corresponding code has been added

**Comment by jhonjamttt** [comment]
@AlexeyAB Hello, I have a great doubt, in the yolov4.cfg file the loss is established in ciou, however during training, it appears in the console iou loss?
![image](https://user-images.githubusercontent.com/76706453/106141596-19514800-614f-11eb-9d38-3f92185afc4a.png)

This means that the loss of ciou is not being used and that the model is using ciou for the regression of the bounding box?

**Comment by folkien** [comment]
what is `v'` in trade-off parameter alpha definition? @AlexeyAB
